charts/airbyte-webapp/templates/deployment.yaml

```diff
@@ -72,6 +72,21 @@ spec:
             configMapKeyRef:
               name: {{ .Release.Name }}-airbyte-env
               key: CONNECTOR_BUILDER_API_HOST
+        - name: AIRBYTE_VERSION
+          valueFrom:
+            configMapKeyRef:
+              name: {{ .Release.Name }}-airbyte-env
```

Comment:
for `airbyte_version` I'd change this to `name: {{ .Values.global.configMapName | default (printf "%s-airbyte-env" .Release.Name) }}`